scripts/variantstore/AOU_GVS_WORKSPACE.md

```diff
@@ -101,30 +130,34 @@ This is done by running the `GvsCreateFilterSet` workflow with the following par
 
 **Note:** This workflow does not use the Terra Entity model to run, so be sure to select `Run workflow with inputs defined by file paths`
 
+Sometimes this workflow will fail because the Gaussians have not cenverged. Dont panic! It can happen to anyone's data!
+The first step in this case will be to adjust the Guassian for the failed step (there are two possible steps: model creation for the SNPS and model creation for the InDels) to a lower number.
+You can then kick off the workflow again. If that still does not work, or you would prefer to not change the number of Guassians, then you can remove a column from the model creation
+TODO: How do you remove a column from the model creation?
```

Comment:
I don't know if we want to include this in the docs, since we don't have a real process for it.

Reply:
like completely skip that a column could be removed? or specifically the TODO?

Reply:
We don't have a real process for how to pick which column (if that's the same as annotation) to remove.